product_docs/docs/epas/11/epas_inst_linux/03_using_a_package_manager_to_install_advanced_server.mdx

```diff
@@ -259,7 +259,6 @@ You can install EDB Postgres Advanced Server on the following operating systems
 -   [RHEL 8](#on-rhel-8)
 -   [RHEL 7](#on-rhel-7)
 -   [SLES 15](#on-sles-15)
--   [SLES 12](#on-sles-12)
```

Comment:
Why are we removing SLES 12? I mean the topic content is still in the file if we scroll down

Reply:
Good catch. I added it back. Not sure why it was being removed.